jf_agent/main.py

```diff
@@ -14,10 +14,10 @@
 
 import requests
 import json
+import dotenv
```

Comment:
Q: Leftover from local dev?

Reply:
Whoops, looks like a double import. We have this import at line 3 on the master branch already.... I think I got my wire crossed a bit in this file

Good catch, I'll remove it